### DISCUSSION TITLE
Welcome to MSFSVoiceAttackPlugin Discussions!

### DISCUSSION THREAD

**Discussion by jamescl604** [jamescl604/MSFSVoiceAttackPlugin#3] [General]
## 👋 Welcome!
 Use Discussions as a place to connect with other members, ask questions, make suggestions.

**Comment by toto3123** [comment]
sorry for my English https://www.deepl.com/translator :)

I congratulate you for this work,
I'm looking for the same plugin but for fsx-se :'(
1- can you guide me to make the necessary changes in your project to make it compatible with fsx-se

ps: i have download visual studio trial

thank you

**Comment by toto3123** [comment]
happy new year

it works yes yes yes with FSX_SE 
I loaded (source code and compiled with vs2019 "Build" in x86 I change Microsoft.FlightSimulator.SimConnect.dll & simconnect.dll and it works with voice attack (x86) ).

the commands "NAV1 SWAP" "NAV2 SWAP" "COM1 SWAP" "COM2 SWAP" works
the other commands how does it work, for example NAV1 is set to 112.00 i want to set nav1 to 111.20 ?

**Comment by jamescl604** [comment]
Thanks for reaching out.

Here is what the install path looks like for my own machine:
C:\Program Files (x86)\VoiceAttack\Apps\MSFSAgent

The key thing is to create a folder under \Apps\ for the add-in

If you place all the files in the zip file into that MSFSAgent folder it should be finding it.

The only other 2 things to check are:

  1.  make sure you're using the 64-bit version of Voice Attack
  2.  in Voice Attack's settings, make sure the "Enable Plugin Support" is checked

If you've tried all that and it's still not showing up in the Plug-in manager let me know.



________________________________

**Reply by fjgaspar** [reply]
I have done all the steps. As I have the 64-bit version of Voice Attack, my install pack is C:\Program Files (x86)\VoiceAttack\Apps\

I have created the folder MSFSAgent folder, and inside it, I have put another MSFSAgent folder, and inside of it, I have put the contents of msfs-agent folder included in zip file.

I have enabled plugins in Voice Attack, in fact, when I run the program, a message Plugins enabled appears. But if I open the plugin manager, it is empty, and if I load the commands of the vda file, an 'Unable to access plugin - pluggin unavailable' message appears in each command.

**Reply by jamescl604** [reply]
You might have too many nested folders.   Try having all the 8 files from the zip in the first MSFSAgent folder. (delete any sub-folders).  So the folder would be: C:\Program Files (x86)\VoiceAttack\Apps\MSFSAgent

**Comment by KevinMicknick** [comment]
Is this still working? I have tried for several days to get the wildcards working. I have on the log "Plugin 'MSFS Agent - v1.0' initialized and get values back from simconnect but can't get anything changed. I wondered if the simconnect API has changed.
